integration-cli/docker_cli_volume_test.go

```diff
@@ -17,6 +17,10 @@ func (s *DockerSuite) TestVolumeCliCreate(c *check.C) {
 	out, _ := dockerCmd(c, "volume", "create", "--name=test")
 	name := strings.TrimSpace(out)
 	c.Assert(name, check.Equals, "test")
+
+	out, _ = dockerCmd(c, "volume", "create", "test2")
+	name = strings.TrimSpace(out)
+	c.Assert(name, check.Equals, "test2")
```

Comment:
Throwing this in there, I feel like there should be a `volume create` with no name. That auto generates the name.

```
λ docker volume create
a0ae260e2588050bc62253da07b786092fa6b505ff784b02a0a9a1754a2ccd57
λ docker volume ls
DRIVER              VOLUME NAME
local               a0ae260e2588050bc62253da07b786092fa6b505ff784b02a0a9a1754a2ccd57
```